### DISCUSSION TITLE
part-ii/2010-132/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#173] [Question comments]
Part II 2010 Riemann Surfaces: Paper 3, Section II, G 

https://questions.tripos.org/part-ii/2010-132/

**Comment by silkca** [comment]
The first two parts of the question are bookwork - "carefully explaining" apparently means giving the definition of each term in the formula.

The third part is also relatively simple - you just need to match the ramification points up and scale by a factor. The nature of the ramification points can be determined from the Riemann-Hurwitz formula. One can explicitly construct $S$ and $T$ given the ramification points and branch points by results from earlier years.
